Makefile

```diff
@@ -1,29 +1,39 @@
 ifeq ($(OS),Windows_NT)
-       PIP = cmd /C .env\\Scripts\\pip.exe --isolated --disable-pip-version-check
+       PIPCMD = cmd /C .env\\Scripts\\pip.exe --isolated --disable-pip-version-check
+       MYPY = .env/Scripts/mypy
+# You can't change pip.exe being in use on Windows, so we'll copy original one and use it
+       COPYPIPCMD = cmd /C .env\\Scripts\\pip-vendored.exe --isolated --disable-pip-version-check
+       ENV = virtualenv
 else
-       PIP = .env/bin/pip
+       PIPCMD = .env/bin/pip
+       MYPY = .env/bin/mypy
+       COPYPIPCMD = $(PIPCMD)
+       ENV = virtualenv -p python3.5
 endif
 
 .PHONY: install test check
 
 default: .env install test
 
 .env:
-	virtualenv -p python3.5 .env
+	$(ENV) .env
+ifeq ($(OS),Windows_NT)
+	cp .env/Scripts/pip.exe .env/Scripts/pip-vendored.exe
+endif
 
-.env/bin/mypy:
-	$(PIP) install mypy-lang
+$(MYPY):
+	$(PIPCMD) install mypy-lang
 
 install-force: .env
-	$(PIP) install . --upgrade
-	$(PIP) install -r requirements.txt --upgrade
+	$(PIPCMD) install . --upgrade
+	$(COPYPIPCMD) install -r requirements.txt --upgrade
```

Comment:
why do you need both `PIPCMD` and `COPYPIPCMD`. Could we just use one or the other on Windows?